company/okrs/2020_q1.md

```diff
@@ -82,6 +82,12 @@
    1. At least four customers are running structural searches and metrics to validate this KR are tracked in [pings](https://docs.sourcegraph.com/admin/pings).
    1. At least four customers are running AND/OR/NOT queries and metrics to validate this KR are tracked in [pings](https://docs.sourcegraph.com/admin/pings).
 
+1. **Transparent and scalable code-host permissions syncing**
+   1. 95th percentile latency of authorization on the read path (i.e. `authzFilter`) is below 10ms.
```

Comment:
@unknwon: Next iteration we should gather metrics for this and send them over in pings